app/templates/_snippets.html

```diff
@@ -138,9 +138,11 @@ <h2 id="readme">README</h2>
         <h5 class="card-title"><span class="fa fa-files-o"></span> {{package.descriptor.title}}</h5>
         <h6>[{{ package.name }}]</h6>
         <div class="panel-body">
+            {% if package.readme %}
```

Comment:
Hmmm, I wonder if we want this kind of validation logic to be server side. ie. server api makes sure that README is empty string not None (if null).

Having "validation logic" in templates gets nasty (and hard to test).